echo/index.js

```diff
@@ -254,13 +261,20 @@ function Client() {
     debug('request options', requestOptions);
 
     request.get(requestOptions, function onResp(err, response, rawBody) {
-      var statusCode = response && response.statusCode
-        ? response.statusCode
-        : 0;
-
-      if (err || parseInt(statusCode / 100) !== 2) {
-        error('[echoClient] queryAnalytics error', { err: err, body: rawBody, statusCode: statusCode });
-        callback(err || 'error response status code: ' + statusCode);
+      if (err) {
+        var errorResponse = {
+          statusCode: 0,
+          message: err.message 
+        };
+        error('[echoClient] queryAnalytics error', { error: errorResponse.message, statusCode: errorResponse.statusCode, body: rawBody });
+        callback(errorResponse);
+      } else if (response && parseInt(response.statusCode / 100) !== 2) {
+        var errorResponse = {
+          statusCode: response.statusCode,
+          message: response.statusMessage
+        };
+        error('[echoClient] queryAnalytics error', { error: errorResponse.message, statusCode: errorResponse.statusCode, body: rawBody });
```

Comment:
split this over multiple lines, its too long

```
error(
  '[echoClient] queryAnalytics error',
  {
	error: errorResponse.message,
	statusCode: errorResponse.statusCode,
	body: rawBody
  }
);
```

do the same for the other errorlog lines, 
if you have linter set it should be doing this for you? whats the line length?

Reply:
Ill check the linting setup and will report back

Reply:
just checked the package json. we pull in jshint but there is no jshintrc file. so I don't think there is any linting